src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_MAX_LENGTH = 10;
+    private K[] keyStore;
+    private V[] valueStore;
+    private int size;
+
+    public StorageImpl() {
+        keyStore = (K[]) new Object[ARRAY_MAX_LENGTH];
+        valueStore = (V[]) new Object[ARRAY_MAX_LENGTH];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < size; i++) {
+            if (resultBoolaen(key, keyStore[i])) {
+                valueStore[i] = value;
+                return;
+            }
+        }
+        keyStore[size] = key;
+        valueStore[size] = value;
+        size++;
     }
 
     @Override
     public V get(K key) {
+        for (int i = 0; i < size; i++) {
+            if (resultBoolaen(key, keyStore[i])) {
+                return valueStore[i];
+            }
+        }
         return null;
     }
 
     @Override
     public int size() {
-        return -1;
+        return size;
+    }
+
+    private boolean resultBoolaen(K key, K keyStore) {
```

Comment:
method name should be verbs. 
```suggestion
    private boolean compareKeys(K key, K keyStore) {
```